js/index.js

```diff
@@ -287,23 +275,29 @@ $(function() {
 
     var scripts = [bundle, wrapped].join("\n");
 
-    var script = iframe.createElement('script');
```

Comment:
I checked every single backend available in try.purescript.org just now... (twice) and everything is working... including Halogen